CHANGELOG.md

```diff
@@ -1,6 +1,17 @@
 # Changelog
 
-## [v0.4.1] - 2024-08-06
+## [v0.5.0] - 2024-08-30
```

Comment:
```suggestion
## [v0.4.2] - 2024-08-30
```
Mind making this a patch release since it's backwards compatible?

Reply:
Yep will do